CMR/python/cmr/auth/token.py

```diff
@@ -106,42 +113,43 @@ def token_manager(options: dict = None):
 # ##############################################################################
 # functions
 
-def token(token_lambdas=None, options=None):
+def token(token_lambdas=None, config:dict = None) -> str:
```

Comment:
style inconsistency: you leave a space between the semicolon and the variable type in other places but not here (```config:dict```)

Reply:
I think this is the style when using a type hint.